[bug] Xenon is Unhealthy when Restart the K8S physical node

**Describe the problem**

Xenon recovers PANIC in the startup process, if Xenon starts failure, it will not be restarted automatically, and it is in an unhealthy state.

```
# kubectl exec -it sample-mysql-2  -c xenon -- bash
bash-5.1$ curl sample-mysql-2.sample-mysql.default:8801
curl: (7) Failed to connect to sample-mysql-2.sample-mysql.default port 8801 after 1 ms: Connection refused
bash-5.1$ curl sample-mysql-1.sample-mysql.default:8801
curl: (52) Empty reply from server
bash-5.1$
```

```
$ kubectl describe pods sample-mysql-2

  Warning  Unhealthy  81s (x380 over 64m)  kubelet, 172.16.6.68  (combined from similar events): Readiness probe failed: xenon.go:49: unexpected error: get.client.error[dial tcp 10.42.1.160:8801: connect: connection refused]

 2022/03/03 11:43:59.853970 common.go:35:       [PANIC]   get.client.error[dial tcp 10.42.1.160:8801: connect: connection refused]
panic:                                          [PANIC]   get.client.error[dial tcp 10.42.1.160:8801: connect: connection refused]

goroutine 1 [running]:
xbase/xlog.(*Log).Panic(0xc000204320, 0x7f5246, 0x2, 0xc00019fd08, 0x1, 0x1)
  /go/src/github.com/radondb/xenon/src/xbase/xlog/xlog.go:164 +0x157
cli/cmd.ErrorOKs(0x86b6c0, 0xc0001180b0)
  /go/src/github.com/radondb/xenon/src/cli/cmd/common.go:35 +0x245
cli/cmd.xenonPingCommandFn(0xc00025a480, 0xa907a0, 0x0, 0x0)
  /go/src/github.com/radondb/xenon/src/cli/cmd/xenon.go:49 +0x94
vendor/github.com/spf13/cobra.(*Command).execute(0xc00025a480, 0xa907a0, 0x0, 0x0, 0xc00025a480, 0xa907a0)
  /go/src/github.com/radondb/xenon/src/vendor/github.com/spf13/cobra/command.go:603 +0x24b
vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xa59480, 0xc000000180, 0xc00019ff78, 0x407ce5)
  /go/src/github.com/radondb/xenon/src/vendor/github.com/spf13/cobra/command.go:689 +0x2d0
vendor/github.com/spf13/cobra.(*Command).Execute(...)
  /go/src/github.com/radondb/xenon/src/vendor/github.com/spf13/cobra/command.go:648
main.main()
  /go/src/github.com/radondb/xenon/src/cli/cli.go:43 +0x31
```
xenon.log
```
2022/03/08 10:43:11.742683 server.go:186:       [PANIC]        server.rpc.start.error[listen tcp: lookup sample-mysql-0.sample-mysql.default on 10.96.0.10:53: no such host
xbase/xrpc.(*Service).Start
        /go/src/github.com/radondb/xenon/src/xbase/xrpc/xrpc.go:54
server.(*Server).Start
        /go/src/github.com/radondb/xenon/src/server/server.go:185
main.main
        /go/src/github.com/radondb/xenon/src/xenon/xenon.go:80
runtime.main
        /usr/local/go/src/runtime/proc.go:225
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:1371]
 2022/03/08 10:43:11.742773 server.go:173:       [ERROR]        server.got.panic[        [PANIC]        server.rpc.start.error[listen tcp: lookup sample-mysql-0.sample-mysql.default on 10.96.0.10:53: no such host
xbase/xrpc.(*Service).Start
        /go/src/github.com/radondb/xenon/src/xbase/xrpc/xrpc.go:54
server.(*Server).Start
        /go/src/github.com/radondb/xenon/src/server/server.go:185
main.main
        /go/src/github.com/radondb/xenon/src/xenon/xenon.go:80
runtime.main
        /usr/local/go/src/runtime/proc.go:225
runtime.goexit
```

**To Reproduce**

Restart the K8S physical node, this situation occurs about one-fifth probability.

**Expected behavior**

we should check this situation automatically and restart Xenon.

**To Reproduce**

**Expected behavior**

- RadonDB MySQL version:
[v2.1.2](https://github.com/radondb/radondb-mysql-kubernetes/milestone/6)

--- a/mysqlcluster/container/xenon.go
+++ b/mysqlcluster/container/xenon.go
@@ -76,7 +76,11 @@ func (c *xenon) getLivenessProbe() *corev1.Probe {
 	return &corev1.Probe{
 		Handler: corev1.Handler{
 			Exec: &corev1.ExecAction{
-				Command: []string{"pgrep", "xenon"},
+				Command: []string{
+					"sh",
+					"-c",
+					"pgrep xenon && xenoncli xenon ping",
+				},
 			},
 		},
 		InitialDelaySeconds: 30,

--- a/mysqlcluster/container/xenon_test.go
+++ b/mysqlcluster/container/xenon_test.go
@@ -96,7 +96,11 @@ func TestGetXenonLivenessProbe(t *testing.T) {
 	livenessProbe := &corev1.Probe{
 		Handler: corev1.Handler{
 			Exec: &corev1.ExecAction{
-				Command: []string{"pgrep", "xenon"},
+				Command: []string{
+					"sh",
+					"-c",
+					"pgrep xenon && xenoncli xenon ping",
+				},
 			},
 		},
 		InitialDelaySeconds: 30,

